### ISSUE TITLE
Running Cookie Bot on new save

### ISSUE THREAD

**Issue by vern1701** [prinzstani/CookieBot#50]
Cookie Bot shows that it's first goal on a new save from scratch is related to the last stage of Grandmapocalypse.  There are numerous goals to attain before that and I am curious why it doesn't go through and actually list these goals as it plays through.  This is not a bug, but I find it odd that a bot on a new save can't list beginning goals or achievements as it plays.  Care to elaborate?

**Comment by DarkPulse** [comment]
There's one main goal and a bunch of sub-goals that the bot tries to go through in any given playthrough

The main goal is basically the goal where as soon as it's achieved, assuming it's not waiting for something else (usually a plant to finish maturing so it can harvest it for its seed), it will ascend and begin a new cycle.

Subgoals are smaller ones, usually not related to a major achievement, which is stuff like trying to buy new upgrades/harvest plants/buy buildings/etc.

For example, my current major achievement is for 1 quindecillion cookies to be baked in one ascension. As soon as this is reached, unless the bot is waiting to harvest a plant, it will ascend. But it's _**ALSO**_ saving 636.677 tredecillion cookies (for bank) at a minimum, waiting to buy Putri Salju (which it never will since I'll hit the major achievement first), trying to grow Golden Clovers and Chimerose on the farm (and if they appear before the major achievement is hit, it will let them grow, harvest them, then ascend), or failing all of those, will hard ascend in 34 days (but it will only take about another day at most to achieve it, really).

Lastly, if the achievement isn't exactly major enough (such as some of the earliest milestones), it will tend to just collect those on the way through to some larger achievement. Ascending, for example, is pointless without at least one prestige level, so it will never select some mundane achievement like "bake 1000 cookies" to ascend for.

Basically, after Hardcore and Neverclick are attained, the bot begins knocking out all the grandma-related achievements. The grandmapocalypse is, naturally, part of that. Once that's done, it will begin doing stuff like getting antimatter condensers, develop the dragon, and get the season switcher. From there, it begins completing all season achievements.

TL;DR: [Read the FAQ.](https://github.com/prinzstani/CookieBot/blob/master/FAQ.md)

**Comment by prinzstani** [comment]
Someone could turn this into some text for the FAQ.

**Comment by prinzstani** [comment]
Added text to the FAQ and added more info to display during playthrough.
